src/sparseml/pytorch/optim/modifier_quantization.py

```diff
@@ -114,6 +117,7 @@ def __init__(
         quantize_embeddings: bool = True,
         reduce_range: bool = False,
         quantize_linear_activations: bool = True,
+        enable_int4_activations: bool = False,
```

Comment:
We also need to add modifier properties here for `activation_qconfig_kwargs` and `weight_qconfig_kwargs` as discussed.